tools/build.js

```diff
@@ -378,13 +379,14 @@ task('Copy Java libraries from BUILD_DIR/lib to APP_HOME/lib.', [], function cop
 
 
 task("Call pmake with JS Maker to build 'foam-bin.js'.", [], function genJS() {
+  setenv();
```

Comment:
curious why do we need setenv() just now?

Reply:
Because no one knows how/where to set FOAM_BIN_VERSION so it works with both -X and not -X.